### PR TITLE
Replace Gogs to Gitea in installation options

### PR DESCRIPTION
**Pull request by joubertredrat**

### CODE REVIEW

**Comment by thibaultmeyer** [comment]
LGTM

File: conf/app.ini [hunk 1]
```diff
@@ -38,7 +38,7 @@ PREVIEWABLE_FILE_MODES = markdown
 [repository.upload]
 ; Whether repository file uploads are enabled. Defaults to `true`
 ENABLED = true
-; Path for uploads. Defaults to `data/tmp/uploads` (tmp gets deleted on gogs restart)
+; Path for uploads. Defaults to `data/tmp/uploads` (tmp gets deleted on gitea restart)
```
Comment (bkcsoft) [inline conf/app.ini:41]: `tmp gets deleted when gitea is restarted` :)

File: conf/app.ini [hunk 2]
```diff
@@ -93,7 +93,7 @@ HTTP_ADDR = 0.0.0.0
 HTTP_PORT = 3000
 ; Permission for unix socket
 UNIX_SOCKET_PERMISSION = 666
-; Local (DMZ) URL for Gogs workers (such as SSH update) accessing web service.
+; Local (DMZ) URL for Gitea workers (such as SSH update) accessing web service.
```
Comment (bkcsoft) [inline conf/app.ini:96]: `accessing the web service`

File: conf/app.ini [hunk 3]
```diff
@@ -120,7 +120,7 @@ MINIMUM_KEY_SIZE_CHECK = false
 OFFLINE_MODE = false
 DISABLE_ROUTER_LOG = false
 ; Generate steps:
-; $ ./gogs cert -ca=true -duration=8760h0m0s -host=myhost.example.com
+; $ ./main cert -ca=true -duration=8760h0m0s -host=myhost.example.com
```
Comment (bkcsoft) [inline conf/app.ini]: `./gitea`

[the remark this answers is not in the document]
Comment (joubertredrat) [reply conf/app.ini]: @andreynering yeah, we will talk about this on https://github.com/go-gitea/docs/issues/3, I will replace here now

**Review by bkcsoft — CHANGES_REQUESTED** [review]
1 error and 2 cleanups :)

**Comment by andreynering** [comment]
LGTM

**Pull request merged**